sides/jollof-rice.md

```diff
@@ -0,0 +1,37 @@
+# Jollof rice
+
+A Ghanaian version of this spicy staple, eat it with some sweet plantains and a protein on the side. This recipe comes from [Tasty](https://tasty.co/recipe/ghanaian-jollof-rice-as-made-by-tei-hammond).
+
+
+## Ingredients:
+
+- 2 large yellow onions, roughly chopped
+- 1/3 cup + 2 tablespoons vegetable oil, divided
+- 2 (14 ounce) cans diced tomato
+- 1 (6 ounce) can tomato paste
+- 1 habanero pepper
+- 2 teaspoons curry powder
+- 1 teaspoon garlic powder
+- 1 teaspoon ground ginger
+- 1/2 teaspoon mixed dried herbs
+- 3 chicken bouillon cubes, crushed
+- 2.5 cups long grain rice, rinsed
+- 1 (12 ounce) bag frozen mixed vegetables
+- 1.5 cups water
+
+
+## Instructions
+
+1. Roughly chop 2 onions into chunks.
+
+2. Add 2 tablespoons of oil to a blender and a chunk of onion and pulse until smooth. Then add another onion chunk and repeat until the onions look like apple sauce. Open a window, as this may sting your eyes. Transfer onion slurry to a medium bowl.
+
+2. Add the 2 cans diced tomatoes, 1 can tomato paste, and habanero pepper to the blender. Pulse until smooth. Transfer to a separate medium bowl.
```

Comment:
Can this step be done while the onion puree is cooking to save time and dirty fewer dishes?